api/views.py

```diff
@@ -212,3 +254,17 @@ def post(self, request):
             return JsonResponse(
                 {"error": "An error has ocurred"}, status=status.HTTP_400_BAD_REQUEST
             )
+
+
+class ChangePasswordView(UpdateAPIView):
+    serializer_class = PasswordSerializer
+    permission_classes = [permissions.IsAuthenticated]
```

Comment:
```suggestion
    permission_classes = [permissions.IsAuthenticated, IsProfileOwner]
```
Mais je sais pas si ce permission ira avec ce `view`

Reply:
Pas tout à fait avec celle-ci car l'endpoint ne spécifie pas un utilisateur particulier. L'URL est : `api/v1/passwordChange`, donc soit l'utilisateur est identifié et donc on change son mot de passe, soit l'utilisateur n'est pas identifié et donc on ne peut rien faire.

`IsProfileOwner` serait nécessaire si par exemple on pouvait spécifier quel profile on veut modifier, genre `api/v1/users/<user_id>/passwordChange`, car dans ce cas il faudrait s'assurer que `user_id` corresponde bien à l'utilisateur identifié. Mais dans notre cas on n'a pas besoin.